### DISCUSSION TITLE
ZEPHYR_TOOLCHAIN_VARIANT not specified and no Zephyr SDK is installed.

### DISCUSSION THREAD

**Discussion by skylinezone** [zephyrproject-rtos/zephyr#65271] [Q&A]
![微信截图_20231116155808](https://github.com/zephyrproject-rtos/zephyr/assets/129871808/e0e4ba0e-1c2b-427e-9924-f1404d081bfd)
![微信截图_20231116155734](https://github.com/zephyrproject-rtos/zephyr/assets/129871808/52c26680-3c4e-4170-9984-23fcb1e7f25d)
Some can help me?

**Comment by ndrs-pst** [comment]
Based on your screen capture, make sure you already have 
C:/Users/13690/zephyr-sdk-0.16.3/arm-zephyr-eabi.

By the way, please refer to this link
https://zephyrproject.org/managing-multiple-versions-of-the-zephyr-rtos/
for some insight (even it's not directly related to your problem)

**Comment by skylinezone** [comment]
Thank you! Already solved the problem by re-starting all the steps in a new virtual environment  ---- Replied Message ***@***.***>Date11/18/2023 00:32 ***@***.***> CcZhang ***@***.***>***@***.***>SubjectRe: [zephyrproject-rtos/zephyr] ZEPHYR_TOOLCHAIN_VARIANT not specified and no Zephyr SDK is installed. (Discussion #65271)
Based on your screen capture, make sure you already have
C:/Users/13690/zephyr-sdk-0.16.3/arm-zephyr-eabi.
By the way, please refer to this link
https://zephyrproject.org/managing-multiple-versions-of-the-zephyr-rtos/
for some insight (even it's not directly related to your problem)

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/zephyrproject-rtos/zephyr/discussions/65271#discussioncomment-7601249",
"url": "https://github.com/zephyrproject-rtos/zephyr/discussions/65271#discussioncomment-7601249",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
